lib/tasks/dev.rake

```diff
@@ -0,0 +1,51 @@
+# Don't add any includes that aren't in the Ruby std-lib here,
```

Comment:
That would be a way to make setting up in travis/vagrant more DRY yes.

However I'm not really convinced that we need to 'hide' away the standard rails env setup tasks from people. People are used to this, this is mentioned in documentation, tutorials etc. etc.

Let's do this for travis/vagrant and only mention this after the 'standard' steps like: 
```
To make this more convenient for you we have setup a rake task that executes the above steps:

bundle exec rake dev:boostrap
```

Okay? :-)